darwin/dataset/split_manager.py

```diff
@@ -269,6 +268,7 @@ def _stratify_samples(
     if test_percentage == 0.0:
         return list(set(X_train.astype(np.int))), list(set(X_tmp.astype(np.int))), []
 
+    print((test_percentage / (val_percentage + test_percentage)))
```

Comment:
If we really want to print something, how feasible/easy would it be to use `rich` here?

Reply:
removed, thanks nice catch :)